ng-dev/pr/common/validation/validations.ts

```diff
@@ -102,3 +106,49 @@ export function assertPendingState(pullRequest: RawPullRequest) {
       throw PullRequestFailure.isMerged();
   }
 }
+
+/**
+ * Assert the pull request has all necessary CLAs signed.
+ * @throws {PullRequestFailure} if the pull request is missing a necessary CLA signature.
+ */
+export function assertSignedCla(pullRequest: PullRequestFromGithub) {
```

Comment:
I guess, now I see why you wanted to store the statuses in a variable. Because now we need to compute them in two assertion functions here. Not really a big deal, but if we'd want to avoid this, then I think it would be reasonable to accept the statuses directly as parameter here. No strong opinion here though. It doesn't matter a lot I suppose.

Reply:
Lets keep it as is, I personally like the fact that you provide the entire `PullRequestFromGithub` object that you are validating the assumption for rather than having to know what parts of a pull request have to be checked for each assertion.

Reply:
👍